st2client/st2client/base.py

```diff
@@ -447,6 +447,12 @@ def _print_client_settings(self, args):
         print("")
         print("Proxy settings:")
         print("---------------")
-        print("HTTP_PROXY: %s" % (os.environ.get("HTTP_PROXY", "")))
-        print("HTTPS_PROXY: %s" % (os.environ.get("HTTPS_PROXY", "")))
+        print(
+            "HTTP_PROXY: %s"
+            % (os.environ.get("http_proxy", os.environ.get("HTTP_PROXY", "")))
+        )
+        print(
+            "HTTPS_PROXY: %s"
+            % (os.environ.get("http_proxy", os.environ.get("HTTPS_PROXY", "")))
```

Comment:
is this http_proxy or http**s**_proxy for HTTPS_PROXY ENV?

Reply:
Good catch. If you submit a PR to add the missing `s`, then I can merge it fairly quickly. Thanks!

Reply:
@cognifloyd  Done, please check: https://github.com/StackStorm/st2/pull/5678.

feel free to let me know if any parts need to be modified.